app/server/lib/ExternalStorage.ts

```diff
@@ -4,6 +4,7 @@ import {createTmpDir} from 'app/server/lib/uploads';
 import {delay} from 'bluebird';
 import * as fse from 'fs-extra';
 import * as path from 'path';
+import {isAffirmative} from 'app/common/gutil';
```

Comment:
tiny lint - we don't enforce it, but most files sort the imports so app/ stuff would go first.